File: exercises/beecrowd/1017/1017.ts
// https://judge.beecrowd.com/pt/problems/view/1017
import * as fs from 'fs';

const input = fs.readFileSync('/dev/stdin', 'utf8');
const lines = input.split(/\r?\n/);

const time = Number(lines.shift());
const speed = Number(lines.shift());

console.log(((time*speed)/12.0).toFixed(3));

File: exercises/beecrowd/1017/1017.go
// https://judge.beecrowd.com/pt/problems/view/1017
package main

import (
	"fmt"
)

func main() {
	var time, speed int

	fmt.Scanln(&time)
	fmt.Scanln(&speed)

	fmt.Printf("%.3f\n", (float64(time)*float64(speed))/12.0)
}
